remote_attestation/rust/src/tests/crypto.rs

```diff
@@ -0,0 +1,217 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::{
+    crypto::{
+        get_sha256, AeadEncryptor, KeyNegotiator, KeyNegotiatorType, SignatureVerifier, Signer,
+        AEAD_ALGORITHM_KEY_LENGTH, CLIENT_KEY_PURPOSE, KEY_AGREEMENT_ALGORITHM_KEY_LENGTH,
+        NONCE_LENGTH, SERVER_KEY_PURPOSE, SHA256_HASH_LENGTH, SIGNATURE_LENGTH,
+        SIGNING_ALGORITHM_KEY_LENGTH,
+    },
+    message::EncryptedData,
+};
+use assert_matches::assert_matches;
+
+// Keys are only used for test purposes.
+// Were created by printing out values in the `crate::crypto`.
```

Comment:
It would be good to state exactly how they were created; how would someone else re-create them (or more) otherwise?

Reply:
I just added `println!` code to print all of the keys.
It's possible to add a feature gated print, but it will look scary for a reviewer: i.e. we are printing out private keys.